usage-updater/lib/Lambda.js

```diff
@@ -0,0 +1,38 @@
+const Rx = require('rx');
+const Config = require('./Config');
```

Comment:
`Config.js` is where the config is for the Lambda. I think I originally intended it to be generated (or pulled from S3) by the deploy script, will go back and update.